manifest.webapp

```diff
@@ -128,5 +128,15 @@
       "trigger": "@event io.cozy.timeseries.geojson:CREATED",
       "debounce": "2m"
     }
+  },
+  "notifications": {
+    "geolocationQuota": {
+      "description": "Notifies the user when is geolocation quota will expire soon or is expired",
+      "collapsible": false,
+      "stateful": true,
+      "multiple": false,
+      "default_priority": "high",
```

Comment:
For my culture, what is the impact of this priority?

Reply:
It is forwarded by the stack to the notification provider. For push notification, it allows to send data only notification that can be _received_ by the app even if notification permission is not granted (but they are not _displayed_).

See https://rnfirebase.io/messaging/usage#message-handlers

> In cases where the message is data-only and the device is in the background or quit, both Android & iOS treat the message as low priority and will ignore it (i.e. no event will be sent). You can however increase the priority by setting the priority to high (Android) and content-available to true (iOS) properties on the payload.

Anyway, we talked about this kind of data only notification for this feature but it is not used yet. So I will remove the high default priority for the moment.